boltdesignsystem/bolt
# DS-443 search results redesign

## Jira

https://pegadigitalit.atlassian.net/browse/DS-443

## Summary

Adds `variant`, `layout`, and `gutter` props to listing teaser; adds Video Thumbnail component.

## Details

1. `variant` prop has been added to handle card style variant (and other variants we might create down the road).
2. `layout` prop has been added to handle vertical and responsive.
3. `gutter` prop has been added to handle the spacing between signifier and content.
4. Video Thumbnail component has been created to handle any kind of video thumbnails that are not video embeds.

## How to test

Run the branch locally and read all the docs on the new props, plus the Search Result List use case. Make sure everything works as expected.

@christophersmith262 This is the code I need you to [review very carefully](https://github.com/boltdesignsystem/bolt/blob/feature/ds-443-search-results-redesign/docs-site/src/pages/pattern-lab/_patterns/40-components/listing-teaser/65-listing-teaser-use-case-search-result-list.twig) and [this schema](https://github.com/boltdesignsystem/bolt/blob/feature/ds-443-search-results-redesign/packages/components/bolt-listing-teaser/listing-teaser.schema.js), check all the `listing-teaser.twig` usage, make sure all the props are set up to be easily implemented with Drupal.
